po/be.po

```diff
@@ -1262,8 +1262,8 @@ msgstr "Для перазапуску \"$(unit)\" патрабуецца аўт
 
 #: src/core/dbus-unit.c:566
 msgid ""
-"Authentication is required to send a UNIX signal to the processes of "
-"'$(unit)'."
+"Authentication is required to send a UNIX signal to the processes of '$"
+"(unit)'."
```

Comment:
Please drop the commit that "updates" the po files. There are no changes except some that lines are wrapped differently. This doesn't seem to be caused by a change in the strings, but instead a different version of the gettext tools. The existing formatting was arguably better.

Reply:
is there some config that can be set? it's annoying having to manually understand what to commit and what to skip

Reply:
I recently run the command, but I did not observe that kind of changes. Also, the line wrap looks quite spurious. I agree with dropping the commit.